tests/modules/vcflib/vcfbreakmulti/test.yml

```
@@ -0,0 +1,19 @@
+- name: vcflib vcfbreakmulti test_vcflib_vcfbreakmulti
+  command: nextflow run tests/modules/vcflib/vcfbreakmulti -entry test_vcflib_vcfbreakmulti -c tests/config/nextflow.config
+  tags:
+    - vcflib/vcfbreakmulti
+    - vcflib
+  files:
+    - path: output/vcflib/test.breakmulti.vcf.gz
+      md5sum: 666d2d970399cf219dc1dc1eadff9170
+    - path: output/vcflib/versions.yml
+      contains: "1.0.3"
```

```suggestion
      contains: ["1.0.3"]
```

This should fix the tests :p

---

Nvm I didn't look at it carefully, you don't have to check for `versions.yml` anymore. The tests will do this automatically